### PR TITLE
fix: TreeSelect removeIcon and clearIcon not working

### PR DESCRIPTION
**Pull request by sosohime**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/18870

### 💡 Background and solution
支持传入 removeIcon 和 clearIcon

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix TreeSelect removeIcon and clearIcon not working.   |
| 🇨🇳 Chinese |    |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/tree-select/index.zh-CN.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ title: TreeSelect
 | showSearch | 是否支持搜索框 | boolean | 单选：`false` \| 多选：`true` |  |
 | size | 选择框大小，可选 `large` `small` | string | 'default' |  |
 | suffixIcon | 自定义的选择框后缀图标 | ReactNode | - | 3.10.0 |
+| removeIcon | 自定义的多选框清除图标 | ReactNode | - | 3.23.5 |
+| clearIcon | 自定义的多选框清空图标 | ReactNode | - | 3.23.5 |
```
Comment (afc163) [inline components/tree-select/index.zh-CN.md]: 加 ts 定义就行了，文档和演示都不暴露。

Comment (sosohime) [reply components/tree-select/index.zh-CN.md]: 不暴露的原因是啥呢，防止低版本产生混淆？

Comment (afc163) [reply components/tree-select/index.zh-CN.md]: 不鼓励去修改设计规范。

**Comment by afc163** [comment]
可以加个测试用例以免改挂。

**Comment by sosohime** [comment]
done

**Pull request merged**